openassessment/xblock/static/sass/oa/elements/_staff.scss

```diff
@@ -17,34 +17,31 @@
     border-top: ($baseline-v/4) solid $color-decorative-staff;
     padding: $baseline-v ($baseline-h/2);
     background: $staff-bg;
+    display: none;
   }
 
   .ui-staff {
 
-    // CASE: area is collapse/expand friendly
-    &.ui-toggle-visibility {
+    .ui-slidable {
 
-      .staff-info__title__copy {
-        @include margin-left(($baseline-h/4));
+      .icon {
```

Comment:
Note to reviewers: we used to inherit this icon CSS from the ui-toggle-visibility stuff. When we get to changing the student views, we may be able to promote some of this CSS out of _staff.scss.